src/ui-test/utils.ts

```diff
@@ -0,0 +1,45 @@
+import { BottomBarPanel, InputBox, WebDriver, Workbench } from "vscode-uitests-tooling";
+
+export const TEST_ARRAY_RUN = [
+    "Routes startup",
+    "Hello Camel from yaml"];
+
+export const CAMEL_RUN_DEBUG_ACTION_LABEL = 'Run Camel Application with JBang and Debug';
+export const CAMEL_RUN_ACTION_LABEL = 'Run Camel Application with JBang';
+export const CAMEL_ROUTE_YAML_WITH_SPACE = 'demo route.camel.yaml';
+/**
+ * Executes a command in the command prompt of the workbench.
+ * @param command The command to execute.
+ * @returns A Promise that resolves when the command is executed.
+ */
+export async function useCommandInPalette(command: string): Promise<void> {
```

Comment:
Could you please try to reuse existing implementation in utils package? https://github.com/djelinek/vscode-uitests-tooling/blob/main/src/components/custom/CommandPalette.ts#L17 ?.. in case its not sufficient enough, skip this time and we will improve later

Reply:
Nice! Thanks, will try

Reply:
We need to investigate it more, now both static methods (open/get) )receive:
`NoSuchElementError: no such element: Unable to locate element: {"method":"css selector","selector":".monaco-quick-open-widget"}` (but open really opens palette)

Reply:
ok.. probably need to update that methods to latest changes on extester side. thanks for investigation